src/components/EventDateFilters.vue

```diff
@@ -38,15 +38,19 @@ export default {
         return forDisplay(this.$store.state.filters.startDate);
       },
       set(value){
-        this.$store.commit('setFilters',{startDate: value})
+        process.nextTick(() => {
+          this.$store.commit('setFilters',{startDate: value})
```

Comment:
(I tried to wrap the contents of the store's `setFilters` method in a call to nextTick, rather than each of these, but that caused an issue where the query string wasn't being updated until the next time the filters changed)